README.md

```diff
@@ -31,11 +30,25 @@ Load the `dragscroll.js` in a preferable way (that is an UMD module):
 <script src="path/to/dragscroll.js"></script>
 ```
 
-Add the `dragscroll` class to a scrollable element:
+Add a style to `dragscroll` class:
+```css
+.dragscroll{
+  overflow: scroll;
+  cursor : grab;
+  cursor : -o-grab;
+  cursor : -moz-grab;
+  cursor : -webkit-grab;
+  overflow-x: hidden;
+  overflow-y: hidden;
+}
+```
 
+Add the `dragscroll` class to a scrollable element:
 ```html
-<div class=dragscroll>
-    Big text goes here...
+<button id="zoom-in" onclick="zoomIn(20);">확대</button>
+<button id="zoom-out" onclick="zoomOut(20);">축소</button>
+<div class="dragscroll">
+  <img id="dragsimg" src="./test.png" />
```

Comment:
Does it only work for images? What happens with text content? (Should be somehow explained in the readme)